src/com/google/javascript/jscomp/RewriteAsyncFunctions.java

```diff
@@ -243,10 +243,35 @@ private void convertAsyncFunction(NodeTraversal t, LexicalContext functionContex
       newBody.addChildToBack(IR.constNode(IR.name(ASYNC_ARGUMENTS), IR.name("arguments")));
       NodeUtil.addFeatureToScript(t.getCurrentFile(), Feature.CONST_DECLARATIONS);
     }
+    boolean needsSuperTranspilation = compiler.getOptions().needsTranspilationFrom(FeatureSet.ES6);
     for (String replacedMethodName : functionContext.replacedSuperProperties) {
+      // MS Edge 17 cannot properly capture references to "super" in an arrow function.
```

Comment:
Do you have any link we could insert here pointing to the MS Edge bug?
We'd really like to be able to track when the MS Edge bug gets fixed, or if it's already fixed and we're just stuck because of installed browsers not getting the fix.

Reply:
I created one and added the link